stream/sentinelone-intel/src/.env.sample

```diff
@@ -0,0 +1,14 @@
+OPENCTI_URL=http://opencti:8080
+OPENCTI_TOKEN=CHANGEME
+CONNECTOR_ID=CHANGEME
+CONNECTOR_NAME=SentinelOne Intel Stream Connector
+CONNECTOR_SCOPE=CHANGEME
+CONNECTOR_LOG_LEVEL=error
+CONNECTOR_LIVE_STREAM_ID=CHANGEME # ID of the live stream created in the OpenCTI UI
+CONNECTOR_LIVE_STREAM_LISTEN_DELETE=true
+CONNECTOR_LIVE_STREAM_NO_DEPENDENCIES=true
+SENTINELONE_INTEL_API_URL=CHANGEME
+SENTINELONE_INTEL_API_KEY=CHANGEME
+SENTINELONE_INTEL_ACCOUNT_ID=CHANGEME_OR_EMPTY
+SENTINELONE_INTEL_SITE_ID=CHANGEME_OR_EMPTY
+SENTINELONE_INTEL_GROUP_ID=CHANGEME_OR_EMPTY
```

Comment:
Please put the same default values as config.yml and in settings.py. And only let the mandatory fields uncommented